packages/navstar/extra/dcos-navstar.service

```diff
@@ -5,7 +5,7 @@ Description=Navstar: A distributed systems & network overlay orchestration engin
 Restart=always
 StartLimitInterval=0
 RestartSec=5
-WorkingDirectory=${PKG_PATH}/navstar
+WorkingDirectory=/opt/mesosphere/active/navstar/navstar
```

Comment:
There is prior art for this in https://github.com/dcos/dcos/blob/master/packages/spartan/extra/dcos-spartan.service